### DISCUSSION TITLE
Criando um App com React, TypeScript, Hooks, Context e Forms - Router e yup

### DISCUSSION THREAD

**Discussion by fabiooroberto** [balta-io/forum#479] [Tira Dúvidas] [answered]
Estava no final do curso, quando quase não consegui completar.
O curso faz referencia ao react-router-dom na versão 5, mas agora ele já se encontra na versão 6.
Pelo que pesquisei, mudou algumas coisas na parte de rotas, aonde ñ existe mais o switch e fui substituído pelo Routes e no hookform/resolvers agora tem que importar: import { yupResolver } from '@hookform/resolvers/yup';

Seria bom ter um vídeo de atualização ou explicação sobre essas mudanças.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @fabiooroberto, como vai?
Muito obrigado pelo feedback 🚀
